src/include/sof/audio/dcblock/dcblock.h

```diff
@@ -11,6 +11,19 @@
 #include <stdint.h>
 #include <sof/platform.h>
 #include <ipc/stream.h>
+#include <sof/compiler_info.h>
+
+/* __XCC__ is both for xt_xcc and xt_clang */
+#if defined(__XCC__)
+#include <xtensa/config/core-isa.h>
+#if XCHAL_HAVE_HIFI4
+#define DCBLOCK_HIFI4
+#else
+#define DCBLOCK_GENERIC
+#endif
+#else
+#define DCBLOCK_GENERIC
+#endif
```

Comment:
How about some indentation?
```C
#if defined(__XCC__)
#  include <xtensa/config/core-isa.h>
#  if XCHAL_HAVE_HIFI4
#    define DCBLOCK_HIFI4
#  else
#    define DCBLOCK_GENERIC
#  endif
#else
#  define DCBLOCK_GENERIC
#endif
```

Reply:
I've never seen `   #define`, I've always seen `#   define` as in my example above.

This manual shows the latter and claims it is ANSI C. It does not say anything about the former.
https://www.gnu.org/software/indent/manual/html_chapter/indent_1.html#Indentation

I recommend staying on the safe side and sticking to the usual syntax. I think it's also more readable.

Reply:
aha, it is the first time I saw #   define, since write as this some how like a marked comment. and I've always seen    #define in some blogs and the below pictures are from C11 standard document chapter 6.10. https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf. 
![Screenshot from 2023-03-09 14-05-41](https://user-images.githubusercontent.com/93516384/223936699-77e8c188-dbc1-4769-9b81-bd9b90c0ef82.png)
![Screenshot from 2023-03-09 14-06-08](https://user-images.githubusercontent.com/93516384/223936713-7b448c57-dc16-4f08-be9c-3dda66d34bce.png)

Reply:
I hesitate to wade into a code style discussion, especially outside of Zephyr, but note that there are a lot of hacky half-preprocessors in the world that do things like look for include dependencies, etc... Let's please not confuse them unnecessarily by putting whitespace at the start of a preprocessor directive line.  That's not a common style (where the post-# whitespace is).

Reply:
> seems we prefer different formats, then I'd like keep the original way to keep style the same with SOF and kernel.


Very good point: let's stay consistent and use `#  define`.

```
$ cd sof
$ git grep -E '^[[:blank:]]+#' '*.c' '*.h' | wc

 => 27 lines. 22 lines after removing some backslash "false positive"

$ git grep -E '^#[[:blank:]]+' '*.c' '*.h' | wc -l

  =>   533  lines
```

The Linux kernel is... not consistent: 14,000 versus 22,000. 

> I've absolutely written "^#" in regular expressions in the past.

+1


PS: I found fortunately no `^[[:blank:]]+#[[:blank:]]+` in SOF.